src/resmom/mom_main.c

```diff
@@ -4755,17 +4275,13 @@ read_config(char *file)
 			return 0;	/* ok for "config" not to be there  */
 	}
 #if !defined(DEBUG) && !defined(NO_SECURITY_CHECK)
-#ifdef	WIN32
-	if (chk_file_sec(file, 0, 0, WRITES_MASK^FILE_WRITE_EA, 0)) {
+	perm = get_permission("write");
+	if (chk_file_sec(file, 0, 0, perm, FULLPATH)) {
```

Comment:
Why does this have to be different between the two ?

Reply:
As per the current design, only for *nix platform we check the full path. And if we see the whole code-base for chk_file_sec(), depending on the file/path/code flow this also varies. 
So to make it consistent with current behavior used the macro